src/ansys/aedt/core/aedt_logger.py

```diff
@@ -931,7 +931,8 @@ def debug(self, msg, *args, **kwargs):
                 msg1 = msg
         else:
             msg1 = msg
-        self._log_on_dekstop(0, msg1, "Global")
+        if not settings.enable_debug_grpc_api_logger:
```

Comment:
Why is that if statement here ? If `settings.enable_debug_grpc_api_logger` is False, you would end up with a log on desktop right ?

Reply:
yes and it would cause a huge slowdown. The debug will be only on the file